### PR TITLE
feat(trace-explorer): Return project for trace root

### PR DESCRIPTION
**Pull request by Zylphrex**
This includes the project for the trace root. It does so while applying a small optimization. Instead of reading the transaction name again while aggregating the traces, we just look at the transactions returned by the breakdown. Also fixes a bug where the breakdowns were being aggregated instead of returning raw events.

### CODE REVIEW

File: src/sentry/api/endpoints/organization_traces.py [hunk 1]
```diff
@@ -350,17 +359,18 @@ def get_traces_breakdowns_query(
...
                 ],
-                orderby=["first_seen()", "last_seen()"],
+                orderby=["precise.start_ts", "precise.finish_ts"],
                 # limit the number of segments we fetch per trace so a single
                 # large trace does not result in the rest being blank
                 limitby=("trace", int(10_000 / len(trace_ids))),
                 limit=10_000,
                 config=QueryBuilderConfig(
-                    functions_acl=["trace_name", "first_seen", "last_seen"],
+                    functions_acl=["first_seen", "last_seen"],
```
Comment (k-fish) [inline src/sentry/api/endpoints/organization_traces.py:373]: do we still need this acl?

File: src/sentry/api/endpoints/organization_traces.py [hunk 2]
```diff
@@ -385,13 +395,12 @@ def get_traces_metas_query(
                     "trace",
                     "count()",
                     # TODO: count if of matching spans
-                    "trace_name()",
                     "first_seen()",
                     "last_seen()",
```
Comment (k-fish) [inline src/sentry/api/endpoints/organization_traces.py:399]: why `precise_*` above but `first_seen` etc. here

**Review by k-fish — APPROVED** [review]

File: src/sentry/api/endpoints/organization_traces.py (hunk 1 above)
Comment (Zylphrex) [reply src/sentry/api/endpoints/organization_traces.py:373]: Good point, it's no longer needed here.

Comment (Zylphrex) [reply src/sentry/api/endpoints/organization_traces.py:373]: Good point, it's no longer needed here.

**Pull request merged**